spec/classes/dns_init_spec.rb

```diff
@@ -403,13 +403,15 @@
         }
       end
 
-      context 'sysconfig', if: ['Debian', 'RedHat'].include?(os_facts[:os]['family']) do
+      context 'sysconfig', if: ['Debian', 'RedHat', 'Ubuntu'].include?(os_facts[:os]['family']) do
         let(:sysconfig_named_path) do
           case facts[:os]['family']
           when 'RedHat'
             '/etc/sysconfig/named'
           when 'Debian'
             '/etc/default/bind9'
+          when 'Ubuntu'
+            '/etc/default/named'
```

Comment:
I think this should make the tests pass:
```suggestion
            facts[:os]['release']['major'] == '18.04' ? '/etc/default/bind9' : '/etc/default/named'
```

Reply:
Thanks for the help. I am not really familiar with Ruby and the testing / mocking here.
Unfortunately this was not "it", yet ...